### PR TITLE
[Merged by Bors] - Make Core Pipeline Graph Nodes Public

### PR DESCRIPTION
**Pull request by cryscan**
# Objective

Make core pipeline graphic nodes, including `BloomNode`, `FxaaNode`, `TonemappingNode` and `UpscalingNode` public.
This will allow users to construct their own render graphs with these build-in nodes.

## Solution

Make them public.
Also put node names into bevy's core namespace (`core_2d::graph::node`, `core_3d::graph::node`) which makes them consistent.

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]

File: crates/bevy_core_pipeline/src/bloom/mod.rs [hunk 1]
```diff
@@ -175,9 +162,9 @@ struct BloomNode {
 }
 
 impl BloomNode {
-    const IN_VIEW: &'static str = "view";
+    pub const IN_VIEW: &'static str = "view";
```
Comment (mockersf) [inline crates/bevy_core_pipeline/src/bloom/mod.rs:165]: if this is going to be public and reused, should it have a more unique name?

Comment (cryscan) [reply crates/bevy_core_pipeline/src/bloom/mod.rs:165]: I'm not sure if this is in the scope of this PR, currently other core nodes all have the same input slot name.

Comment (komadori) [reply crates/bevy_core_pipeline/src/bloom/mod.rs:165]: I think it makes sense that the input slots have the same name if they serve the same purpose for their respective nodes.

**Review by komadori — APPROVED** [review]

**Review by ickk — APPROVED** [review]

**Pull request closed without merging**